### PR TITLE
Fix #12947

### LINKED ISSUE #12947: Parameter 'message' must be a string in FileValidator
When i try to validate file upload, and the file exceeds the 'maxSize' specified in the FileValidator i get "Parameter 'message' must be a string" exception.
### Code

```
$form = new Form();
$file = new File('test');

$file->addValidator(new FileValidator([
	'maxSize'           => '3M',
	'allowEmpty'        => true,
	'allowedTypes'      => [
		'image/jpeg',
		'image/png'
	],
]));

$form->add($file);

if($this->request->isPost())
{
	if($form->isValid(array_merge($this->request->getPost(), $_FILES)))
	{

	}
	else
	{
		foreach($form->getMessages() as $message)
		{
			$this->flash->error($message);
		}
	}
}

$this->view->form = $form;
```

```
{{ form('', 'method': 'post', 'enctype': 'multipart/form-data') }}
    {{ form.render('test') }}
    <input type="submit" value="Go" />
{{ endForm() }}
```

Output

```
Parameter 'message' must be a string
#0 [internal function]: Phalcon\Validation\Message->__construct(false, 'test', 'FileIniSize', NULL)
#1 [internal function]: Phalcon\Validation\Validator\File->validate(Object(Phalcon\Validation), 'test')
#2 [internal function]: Phalcon\Validation->validate(Array, NULL)
#3 /home/be4i/Projects/phalcontest/app/controllers/IndexController.php(29): Phalcon\Forms\Form->isValid(Array)
#4 [internal function]: IndexController->indexAction()
#5 [internal function]: Phalcon\Dispatcher->callActionMethod(Object(IndexController), 'indexAction', Array)
#6 [internal function]: Phalcon\Dispatcher->_dispatch()
#7 [internal function]: Phalcon\Dispatcher->dispatch()
#8 /home/be4i/Projects/phalcontest/public/index.php(42): Phalcon\Mvc\Application->handle()
#9 {main}
```

### Details

* Phalcon version: 3.2.0
* PHP Version: PHP 7.0.20-1~dotdeb+8.2
* Operating System: Debian Jessie
* Installation type: installing via package manager
* Zephir version (if any): 0.9.8-6335775f25
* Server: Apache2

### PR DESCRIPTION
**Pull request by Zaszczyk**
Hello!

* Type: bug fix
* Link to issue: https://github.com/phalcon/cphalcon/issues/12947

**In raising this pull request, I confirm the following:**

- [x] I have read and understood the [Contributing Guidelines](https://github.com/phalcon/cphalcon/blob/master/CONTRIBUTING.md)
- [x] I have checked that another pull request for this purpose does not exist.
- [ ] I wrote some tests for this PR.

Small description of change: I added missing replacePairs

### CODE REVIEW

**Comment by sergeyklay** [comment]
@Zaszczyk Can you create a small test which will cover code you have changed?

**Comment by Zaszczyk** [comment]
@sergeyklay ok, I will try

File: tests/unit/Validation/Validator/FileTest.php [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+                                    '_message' => 'Field file must be an smaller than 1MB',
+                                    '_field'   => 'file',
+                                    '_code'    => 0,
+                                ]
+                            )
+                        ]
+                    ]
+                );
+
+                expect($expectedMessages)->equals($messages);*/
```
Comment (sergeyklay) [inline tests/unit/Validation/Validator/FileTest.php]: What is it?

Comment (Zaszczyk) [inline tests/unit/Validation/Validator/FileTest.php]: I have no possibility to setup phalcon environment locally, so I run tests in travis

Comment (sergeyklay) [inline tests/unit/Validation/Validator/FileTest.php]: Did you try https://github.com/phalcon/box :)

**Comment by Jurigag** [comment]
@sergeyklay we already have this discussion, currently it's not possible to write tests for File validator beacause you can't mockup php functions in zephir beacause zephir ignore namespack fallback policy.

**Comment by sergeyklay** [comment]
@Zaszczyk @Jurigag I'll take a look a bit later. Give me please a few days guys

**Comment by Jurigag** [comment]
I was already doing this https://github.com/Jurigag/cphalcon/blob/2a889c98664d26ed9036b1e0da5201879d05a6f9/tests/unit/Validation/Validator/FileTest.php but it wasn't just working, zephir was just ignoring mocked up functions and was calling php anyway. Only way i see is too move php calls to methods and just extend file validator and override methods for tests but this is kind of bad.

**Comment by Jurigag** [comment]
Remove FileTest since we can't mockup php functions in `Phalcon\Validation\Validator\File` and merge it, just zephir ignores namespace fallback policy so it's impossible. Well there is a stupid way to make it possible - instead of calling functions, move them to methods and call methods, and then in php extend this validator and test this extended validator with overrided methods as mocked functions.

**Comment by Zaszczyk** [comment]
I've added changelog and deleted class FileTest

**Comment by sergeyklay** [comment]
Thank you

**Pull request merged**